pkg/integrations/v4/logs/cfg.go

```diff
@@ -647,6 +648,7 @@ func newNROutput(cfg *config.LogForward) FBCfgOutput {
 		CABundleFile:      cfg.ProxyCfg.CABundleFile,
 		CABundleDir:       cfg.ProxyCfg.CABundleDir,
 		ValidateCerts:     cfg.ProxyCfg.ValidateCerts,
+		Retry_Limit:       cfg.RetryLimit,
```

Comment:
[⚠️] Please note you also need to modify the `cfg_template.go` file to include the `Retry_Limit` option in the `[OUTPUT]` section, after [this line](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/integrations/v4/logs/cfg_template.go#L123). Otherwise, the resulting Fluent Bit configuration will not include the `Retry_Limit` option.